test/unit/warp/src/All.cpp

```diff
@@ -70,46 +70,59 @@ TEMPLATE_LIST_TEST_CASE("all", "[warp]", alpaka::test::TestAccs)
     using Dim = alpaka::Dim<Acc>;
     using Idx = alpaka::Idx<Acc>;
 
-    auto const platform = alpaka::Platform<Acc>{};
-    auto const dev = alpaka::getDevByIdx(platform, 0);
-    auto const warpExtents = alpaka::getWarpSizes(dev);
-    for(auto const warpExtent : warpExtents)
+    if constexpr(alpaka::accMatchesTags<
+                     Acc,
+                     alpaka::TagCpuSycl,
+                     alpaka::TagGpuSyclIntel,
+                     alpaka::TagFpgaSyclIntel,
+                     alpaka::TagGenericSycl>)
     {
-        auto const scalar = Dim::value == 0 || warpExtent == 1;
-        if(scalar)
-        {
-            alpaka::test::KernelExecutionFixture<Acc> fixture(alpaka::Vec<Dim, Idx>::all(4));
-            REQUIRE(fixture(AllSingleThreadWarpTestKernel{}));
-        }
-        else
+        std::cout << "Test disabled for SYCL\n";
```

Comment:
Would it be better to use
```suggestion
        WARN("Test disabled for SYCL");
```
instead ?

It prints a much bigger message:
```
-------------------------------------------------------------------------------
all - alpaka::test::TestAccs - 2
-------------------------------------------------------------------------------
/home/fwyzard/src/alpaka-group/alpaka/test/unit/warp/src/All.cpp:67
...............................................................................

/home/fwyzard/src/alpaka-group/alpaka/test/unit/warp/src/All.cpp:80: warning:
  Test disabled for SYCL

```

Reply:
`WARN` is better because it shows, which test is disabled. Maybe in a future PR, we can enable that warnings will be displayed if `ctest` is executed :thinking: At the moment, if `ctest` is executed, you will not see, that the test is skipped.

Reply:
OK, I'll make the change.